libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/view/MapboxRecenterButton.kt

```diff
@@ -89,16 +92,16 @@ class MapboxRecenterButton : ConstraintLayout {
     fun showTextAndExtend(duration: Long) {
```

Comment:
```suggestion
    fun showTextAndExtend(text: String, duration: Long) {
```

Reply:
This will ensure user can change the text anytime they want and not only one time through the XML. It would also remove storing the state `text` above.

Reply:
Same changes apply for `RouteOverviewButton` and `MapboxSoundButton`